app/models/planning_application.rb

```diff
@@ -303,7 +303,7 @@ def documents_for_decision_notice
   end
 
   def received_at
-    Time.next_immediate_business_day(created_at)
+    super || (Time.next_immediate_business_day(created_at) if created_at)
```

Comment:
I think `created_at` should still always be present?

Reply:
Yeah left it out but then on planning application create page it was calling this code `Time.next_immediate_business_day(created_at)` and nil was blowing up. Maybe I can edit that [method](https://github.com/unboxed/bops/blob/main/config/initializers/business_time.rb#L13-L15) directly though just to return if the argument is nil 

What do you reckon?

Reply:
Ah, I hadn't thought of new records! I think it's probably best as you've already done it, with the check in the model.